### ISSUE TITLE
Rerun terraform plan on deployed svpc_service_project module failed

### ISSUE THREAD

**Issue by xingao267** [terraform-google-modules/terraform-google-project-factory#524]
Step to reproduce:

1. Deploy a VPC host project first

2. Deploy the following service project
```
module "project" {
  source  = "terraform-google-modules/project-factory/google//modules/svpc_service_project"
  version = "~> 10.0.1"

  name                    = "my-vpc-service-project"
  org_id                  = ""
  folder_id               = "xxx"
  billing_account         = "xxx"
  lien                    = true
  default_service_account = "keep"

  shared_vpc    = "my-vpc-host-project"
  activate_apis = []
}
```
It succeeds

3. Rerun `terraform plan` or `terraform apply` on the deployed `svpc_service_project` submodule gives you the following error:

```
Error: Invalid count argument

  on .terraform/modules/project/modules/shared_vpc_access/main.tf line 77, in resource "google_project_iam_member" "gke_host_agent":
  77:   count   = local.gke_shared_vpc_enabled && var.enable_shared_vpc_service_project ? 1 : 0

The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.


Error: Invalid count argument

  on .terraform/modules/project/modules/shared_vpc_access/main.tf line 89, in resource "google_project_iam_member" "gke_security_admin":
  89:   count   = local.gke_shared_vpc_enabled && var.grant_services_security_admin_role ? 1 : 0

The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.
```

**Comment by xingao267** [comment]
Removing the empty `activate_apis ` var seems fix it.

**Comment by bharathkkb** [comment]
@xingao267 is this with tf 0.13 or 0.14?

**Comment by xingao267** [comment]
It's with tf 0.13.5.

**Comment by shako92** [comment]
I experienced the same issue attaching logs.

`Terraform v0.13.5
Configuring remote state backend...
Initializing Terraform configuration...
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

data.google_billing_account.development-billing-account: Refreshing state... [id=01F2D0-E8481D-AD2722]
module.create-org-folders.google_folder.folders[0]: Refreshing state... [id=folders/944190572786]
module.create-org-folders.google_folder.folders[1]: Refreshing state... [id=folders/1084370314591]
module.create-development-projec_iam_bindings.google_project_iam_member.project_iam_additive["default--roles/compute.admin--group:gcp-organization-admins@revolvergaming.com"]: Refreshing state... [id=development-e42f/roles/compute.admin/group:gcp-organization-admins@revolvergaming.com]
module.create-development-project.module.project-factory.random_id.random_project_id_suffix: Refreshing state... [id=5C8]
module.create-development-project.module.project-factory.google_project.main: Refreshing state... [id=projects/development-e42f]
module.create-development-project.module.project-factory.module.project_services.google_project_service.project_services["container.googleapis.com"]: Refreshing state... [id=development-e42f/container.googleapis.com]
module.create-development-project.module.project-factory.module.project_services.google_project_service.project_services["compute.googleapis.com"]: Refreshing state... [id=development-e42f/compute.googleapis.com]
module.create-development-project.module.project-factory.google_service_account.default_service_account: Refreshing state... [id=projects/development-e42f/serviceAccounts/project-service-account@development-e42f.iam.gserviceaccount.com]
module.create-development-project.module.project-factory.google_project_default_service_accounts.default_service_accounts[0]: Refreshing state... [id=projects/development-e42f]
module.create-development-project-tf-sa.google_service_account.service_accounts["project-tf-sa"]: Refreshing state... [id=projects/development-e42f/serviceAccounts/project-tf-sa@development-e42f.iam.gserviceaccount.com]
module.create-development-project-tf-sa.data.template_file.keys["project-tf-sa"]: Refreshing state... [id=e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855]

Error: Invalid count argument

  on .terraform/modules/create-development-project/modules/shared_vpc_access/main.tf line 77, in resource "google_project_iam_member" "gke_host_agent":
  77:   count   = local.gke_shared_vpc_enabled && var.enable_shared_vpc_service_project ? 1 : 0

The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.


Error: Invalid count argument

  on .terraform/modules/create-development-project/modules/shared_vpc_access/main.tf line 89, in resource "google_project_iam_member" "gke_security_admin":
  89:   count   = local.gke_shared_vpc_enabled && var.grant_services_security_admin_role ? 1 : 0

The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.`

**Comment by xingao267** [comment]
It looks it also fails on the main module `terraform-google-modules/project-factory/google` in addition to the `svpc_service_project` submodule.

**Comment by xingao267** [comment]
This seems fixed by TF 0.14.
